http_prompt/commandio.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import six
+
+from .printer import Printer
+from .utils import strip_color_codes
+
+
+class CommandIO():
```

Comment:
The existence of `CommandIO` class seems unnecessary. It wraps a file-like object and provides an exact same interface (`write`, `close`, and `read` etc). Why not just use the file-like object directly? ([BytesIO](https://docs.python.org/3/library/io.html#io.BytesIO), [file object](https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files), [sys.stdout](https://docs.python.org/3.5/library/sys.html#sys.stdout) are all file-like object)